Pyrado/pyrado/environments/mujoco/assets/openai_ant.xml

```diff
@@ -0,0 +1,81 @@
+<mujoco model="ant">
+  <compiler angle="degree" coordinate="local" inertiafromgeom="true"/>
+  <option gravity="0 0 -[gravity]" integrator="RK4" timestep="0.01" wind="[wind_x] [wind_y] [wind_z]"/>
+  <custom>
+    <numeric data="0.0 0.0 0.55 1.0 0.0 0.0 0.0 0.0 1.0 0.0 -1.0 0.0 -1.0 0.0 1.0" name="init_qpos"/>
+  </custom>
+  <default>
```

Comment:
please add empty lines between different things (e.g. assets, bodies, compiler, option,...) like in openai_hopper.py. this makes reading way easier